lib/hipmer/hipmerUtils.py

```diff
@@ -107,7 +106,7 @@ def _validate_inputs(self, params):
 
     def _check_if_authorized_expert(self):
         if self.username not in self.authorized_expert_usernames:
-            raise Exception("You entered a GBP limit > 500 and you are not an authorized expert user. If you believe you should be an expert user, please contact KBase")
+            raise Exception(f"You entered a GBP limit > 500 and your username {self.username} is not an authorized expert user. If believe you should be an expert user, please contact KBase.")
```

Comment:
Was that "you" removed on purpose?  It now reads "If believe you should be".  Read well to me the original way (with the added punctuation).  +1 on the username addition.

Reply:
@mikacashman Thanks! Fixed the typo